pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -98,13 +99,52 @@ func fixKvmEnabled() error {
 	return nil
 }
 
+type virCapabilitiesGuest struct {
```

Comment:
Perhaps use this library https://github.com/libvirt/libvirt-go-xml/blob/master/capabilities.go#L293